libs/core/src/lib/quick-view/quick-view-group-item-content/quick-view-group-item-content.component.ts

```diff
@@ -1,11 +1,31 @@
-import { ChangeDetectionStrategy, Component } from '@angular/core';
+import { AfterViewInit, ChangeDetectionStrategy, Component, ElementRef } from '@angular/core';
+
+import { QuickViewGroupItemContentElementDirective } from './quick-view-group-item-content-element.directive';
 
 @Component({
     selector: 'fd-quick-view-group-item-content',
     templateUrl: './quick-view-group-item-content.component.html',
-    host: {
-        '[class.fd-input]': 'true'
-    },
     changeDetection: ChangeDetectionStrategy.OnPush
 })
-export class QuickViewGroupItemContentComponent {}
+export class QuickViewGroupItemContentComponent implements AfterViewInit {
+    /** @hidden */
+    constructor(private readonly _elRef: ElementRef) {}
+
+    /** @hidden */
+    ngAfterViewInit(): void {
+        this._bindElementAttributes();
+    }
+
+    /** @hidden
+     * Needed for binding the id of the element and id of the proper label to aria-labelledby. */
+    private _bindElementAttributes(): void {
+        const parentId = this._elRef.nativeElement?.parentElement?.id;
```

Comment:
`this._elRef.nativeElement?.parentElement?.id;` looks unsafe. 
Are we eligible to retrieve some attribute from a parent using DOM hierarchy?

Reply:
what if somebody adds a wrapper?